app/experimenter/visualization/api/v3/views.py

```diff
@@ -139,7 +141,7 @@ def process_data_for_consumption(overall_data, weekly_data, experiment):
     return results, other_metrics
 
 
-def generate_results_object(data, experiment):
+def generate_results_object(data, experiment, window="overall"):
```

Comment:
looks like `"overall"` could be a constant somewhere? but i see a bunch of constants littered around so maybe we can clean all that up in a later pass 🤷‍♀️